app/session.py

```diff
@@ -0,0 +1,90 @@
+import uuid
+
+import flask
+import redis
+
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
+
+import itsdangerous
+from flask.sessions import SessionMixin, SessionInterface
+from werkzeug.datastructures import CallbackDict
+
+SESSION_PREFIX = "session"
+
+
+class ServerSession(CallbackDict, SessionMixin):
+    def __init__(self, initial=None, sid=None):
+        def on_update(self):
+            self.modified = True
+
+        CallbackDict.__init__(self, initial, on_update)
```

Comment:
I compare this implementation with `SecureCookieSession` provided by default and they don't seem to use `CallbackDict`, instead they just call `super(SecureCookieSession, self).__init__(initial, on_update)`,  do you know why we need to use `CallbackDict` here?

Reply:
`SecureCookieSession` uses `CallbackDict` behind the scenes. We just skip `SecureCookieSession` because we've got a different implementation